Add changelog entries for blob refactor (#16199)
sourcegraph/sourcegraph: CHANGELOG.md

- Password reset link expiration can be customized via `auth.passwordResetLinkExpiry` in the site config. [#13999](https://github.com/sourcegraph/sourcegraph/issues/13999)
- Campaign steps may now include environment variables from outside of the campaign spec using [array syntax](http://docs.sourcegraph.com/campaigns/references/campaign_spec_yaml_reference#environment-array). [#15822](https://github.com/sourcegraph/sourcegraph/issues/15822)
- Extensions can now add decorations to files on the sidebar tree view and tree page through the experimental `FileDecoration` API. [#15833](https://github.com/sourcegraph/sourcegraph/pull/15833)

> - The extension API now has a dedicated GraphQL method. Extensions previously had to execute an untyped command to interact with the Sourcegraph GraphQL API. [#15566](https://github.com/sourcegraph/sourcegraph/pull/15566)

Reviewer: I'd keep this positive and focus on the value this provides
```suggestion
- Extensions can now easily query the Sourcegraph GraphQL API through a dedicated API method. [#15566](https://github.com/sourcegraph/sourcegraph/pull/15566)
```

Author: Thanks, sounds a lot better

- Syntax highlighting on files with mixed extension case (e.g. `.CPP` vs `.cpp`) now works as expected. [#11327](https://github.com/sourcegraph/sourcegraph/issues/11327)
- After applying a campaign, some GitLab MRs might have had outdated state shown in the UI until the next sync with the code host. [#16100](https://github.com/sourcegraph/sourcegraph/pull/16100)
- The web app no longer sends stale text document content to extensions. [#14965](https://github.com/sourcegraph/sourcegraph/issues/14965)

> - The blob viewer now supports multiple decorations per line as intended. [#15063](https://github.com/sourcegraph/sourcegraph/issues/15063)

Reviewer: Shouldn't this be under "Fixed"? See last release

Author: ![Screenshot from 2020-11-26 21-43-17](https://user-images.githubusercontent.com/37420160/100404865-6a9a6980-3030-11eb-88c8-ca2f8e6a8cad.png)
